suite-native/formatters/src/components/FiatAmountFormatter.tsx

```diff
@@ -14,10 +14,17 @@ type FiatAmountFormatterProps = FormatterProps<string | null> &
     TextProps & {
         network?: NetworkSymbol;
         isDiscreetText?: boolean;
+        isForcedDiscreetMode?: boolean;
     };
 
 export const FiatAmountFormatter = React.memo(
-    ({ network, value, isDiscreetText = true, ...textProps }: FiatAmountFormatterProps) => {
+    ({
+        network,
+        value,
+        isDiscreetText = true,
```

Comment:
nit: Put the prop with default value to the end please.

Reply:
4a845d4e24